modules/about-oadp-update-channels.adoc

```diff
@@ -9,23 +9,25 @@
 
 When you install an OADP Operator, you choose an _update channel_. This channel determines which upgrades to the OADP Operator and to Velero you receive. You can switch channels at any time.
 
-There are three update channels:
+The available update channels:
```

Comment:
Not clear why this should be a fragment, not a complete sentence. I'd go with "The available update channels are" .  I understand, o course, why you removed "three".